src/applications/disability-benefits/all-claims/tests/pages/form0781/traumaticEventsPages.unit.spec.jsx

```diff
@@ -106,9 +106,9 @@ describe('Traumatic Events Pages', () => {
   it('should render mentalHealthSupportAlert in ContentBeforeButtons', () => {
     const { container } = render(<div>{ContentBeforeButtons}</div>);
     const { textContent } = container;
-    const expectedText =
-      'We understand that some of the questions may be difficult to answer.';
-
+    // const expectedText =
+    // 'We understand that some of the questions may be difficult to answer.';
+    const expectedText = `We can connect you with free, confidential support for mental health care or military sexual trauma anytime. We can connect you with support even if you don’t file a claim or disability compensation or you aren’t eligible for compensation.`;
```

Comment:
Make necessary update to content related to https://github.com/department-of-veterans-affairs/vets-website/pull/38023/files#r2246368161.